charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -441,6 +446,13 @@ spec:
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
+      - name: reporting-operator-trusted-ca-bundle
+        valueFrom:
+          configMapKeyRef:
+            name: reporting-operator-trusted-ca-bundle
+            key: data["ca-bundle.crt"]
```

Comment:
```suggestion
        configMap:
          name: reporting-operator-trusted-ca-bundle
          items:
            - key: ca-bundle.crt
              path: ca-bundle.crt
```

Reply:
https://kubernetes.io/docs/concepts/storage/volumes/#configmap

Reply:
wouldn't the key still be data?

Reply:
Nah, this needs the configMapKeyRef